src/Routes/Routes.tsx

```diff
@@ -104,10 +103,20 @@ export const Routes = memo(() => {
       appRoutes.map(route => {
         const MainComponent = route.main
 
+        if (isMobile && !state.isConnected) {
+          const to = {
+            pathname: '/connect-wallet',
+            search: `returnUrl=${location?.pathname ?? '/trade'}`,
+          }
+
+          // eslint-disable-next-line react-memo/require-usememo
```

Comment:
Can we add a comment on why we want to disable this?

Reply:
Absolutely, don't feel like we need to add more commentary for "this is already within a useMemo" lint disables anymore at this point tbh, feels like fluff, but on the other hand doesn't hurt to!
4dc86a56d0

Reply:
Oh that's fair, no real need then. Often in the context of a review I don't see enough of the diff to know that it's in a `useMemo` already!